### DISCUSSION TITLE
Combining raw images

### DISCUSSION THREAD

**Discussion by MatessakCZE** [immich-app/immich#6981] [Feature Request]
### The feature

Hey, Will you ever add support for combining raw and normal images in the preview into one like in google photos? Thanks!
![image](https://github.com/immich-app/immich/assets/77834293/3d656b5b-8ff4-42a8-8ea3-0f31bcf139e4)
![image](https://github.com/immich-app/immich/assets/77834293/cee9b56c-41ef-42cd-b6d4-f8437c2c841e)
![image](https://github.com/immich-app/immich/assets/77834293/55d609c0-c772-432b-82e6-f2807e33377b)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #2479
